andela/team-odd-bn-backend
# #170476231 Improve current implementation of social login

#### What does this PR do?
The PR helps different users to authenticate by using their current Facebook and Google account

#### Description of Task to be completed?
**Current implementation**
It authenticates only the owner of the app.
**Expected**
It should authenticate every user to login via Facebook and Google.

#### How should this be manually tested?
- Clone this repository using the command: `git clone https://github.com/andela/team-odd-bn-backend.git`
- `git clone https://github.com/andela/team-odd-bn-backend.git`
- `cd team-odd-bn-backend`
- git checkout -b ch-improve-social-login-170476231
- `npm install`
- Make sure your `.env` file is updated.
- Drop existing migration `npm run dropTables`
- Update migration `npm run db-migrate`
- Populate the seed data `npm run db-seed-dev`
- `npm run dev-start` check API in development 

#### Any background context you want to provide?
This PR is branched off from [PR #55](https://github.com/andela/team-odd-bn-backend/pull/55). #55 solves some bugs we had.

#### What are the relevant pivotal tracker stories?
[#170476231](https://www.pivotaltracker.com/story/show/170476231)

#### Screenshots (if appropriate)
Login by Facebook
<img width="1440" alt="Screen Shot 2019-12-31 at 14 56 35" src="https://user-images.githubusercontent.com/56690376/71622372-f7009a00-2bdd-11ea-9b1e-029596ade4b2.png">

Login by Google
<img width="1440" alt="Screen Shot 2019-12-31 at 14 57 26" src="https://user-images.githubusercontent.com/56690376/71622377-fc5de480-2bdd-11ea-85f7-0ee8d71ff92d.png">
 
#### Questions:
